api/src/types/s2-api.ts

```diff
@@ -13,21 +13,52 @@ interface S2ApiSuccess {
   status: 200;
 }
 
-export interface S2ApiPaper {
-  abstract: string;
-  arxivId: string | null;
-  authors: S2ApiAuthor[];
-  doi: string;
-  title: string;
+export interface S2Api {
+  paper: S2ApiPaper;
+}
+
+interface S2ApiPaper {
+  id: string;
+  title: S2ApiTitle;
+  paperAbstract: S2ApiAbstract;
+  authors: S2ApiAuthor[][];
```

Comment:
Is `authors` really a list of lists of `S2ApiAuthor`? Or is it just a list of type `S2ApiAuthor`?

Reply:
![Annotation 2020-05-19 155413](https://user-images.githubusercontent.com/52334652/82386353-0d6e8d00-99e9-11ea-9964-10c348103e9f.png)

all authors are contained in a list with a single item, that's just the way this api is :/